<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createOrderWindow</class>
 <widget class="QDialog" name="createOrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Προσθήκη Προϊόντος</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="addItem">
     <property name="text">
      <string>Προσθήκη</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTableWidget" name="productList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Μονάδα Μέτρησης</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Όνομα Προϊόντος</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addItem</sender>
   <signal>clicked()</signal>
   <receiver>createOrderWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>658</x>
     <y>603</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
